src/makeRobotAccountant.js

```diff
@@ -5,8 +5,20 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let count = 0;
+
+  return function(b) {
```

Comment:
use arow function declaration)

Reply:
when you need to return a function you should return the name of the variable which contains the function or it should be the arrow function without naming)